docs/assembling_twir.md

```diff
@@ -54,12 +52,12 @@ These are collected using GitHub queries:
 
 ## Events
 
-[@nellshamrell](https://github.com/nellshamrell) currently covers this section, but would welcome help from additional editors!
+[@andrewpollack](https://github.com/andrewpollack) currently covers this section, but would welcome help from additional editors!
```

Comment:
```suggestion
[@andrewpollack](https://github.com/andrewpollack) and @mariannegoldin](https://github.com/mariannegoldin) currently cover this section, but would welcome help from additional editors!
```

Reply:
Absolutely! Merging this commit (with a minor edit on the suggestion formatting) :raised_hands: